From 8260728d5345174cb8019f3e2a3247e12bf6090d Mon Sep 17 00:00:00 2001
From: Ryo Nakano <ryonakaknock3@gmail.com>
Date: Sun, 9 Apr 2023 19:15:45 +0900
Subject: [PATCH] Metainfo: Use screenshots taken on GNOME

---
 data/reco.metainfo.xml.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/reco.metainfo.xml.in b/data/reco.metainfo.xml.in
index 4eceeca..3df3db2 100644
--- a/data/reco.metainfo.xml.in
+++ b/data/reco.metainfo.xml.in
@@ -26,17 +26,17 @@
   <screenshots>
     <screenshot type="default">
       <caption>Welcome view; perform recording configuration</caption>
-      <image>https://raw.githubusercontent.com/ryonakano/reco/4.0.1/data/screenshots/pantheon/welcome-init-light.png</image>
+      <image>https://raw.githubusercontent.com/ryonakano/reco/4.0.1/data/screenshots/gnome/welcome-init-light.png</image>
     </screenshot>
 
     <screenshot>
       <caption>Recording view</caption>
-      <image>https://raw.githubusercontent.com/ryonakano/reco/4.0.1/data/screenshots/pantheon/recording-light.png</image>
+      <image>https://raw.githubusercontent.com/ryonakano/reco/4.0.1/data/screenshots/gnome/recording-light.png</image>
     </screenshot>
 
     <screenshot>
       <caption>An indication that recording is successfully finished</caption>
-      <image>https://raw.githubusercontent.com/ryonakano/reco/4.0.1/data/screenshots/pantheon/welcome-success-light.png</image>
+      <image>https://raw.githubusercontent.com/ryonakano/reco/4.0.1/data/screenshots/gnome/welcome-success-light.png</image>
     </screenshot>
   </screenshots>
 
-- 
2.34.1

